src/Umbraco.Cms.Persistence.SqlServer/Umbraco.Cms.Persistence.SqlServer.csproj

```diff
@@ -20,4 +20,10 @@
     </AssemblyAttribute>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning">
```

Comment:
This should only be in solution directory Directory.Build.props

Reply:
Stupid NuGet package manager 🙈  I'll fix that, good catch 💪

Reply:
one to watch out for as without privateassets we will leak it downstream